### ISSUE TITLE
go get ./...: exit status 1

### ISSUE THREAD

**Issue by sudo-suhas** [actions/setup-go#68]
When I try to run `go get ./...`, it fails with `exit status 1`. I am not able to figure out what could be the reason.

Workflow config: https://github.com/matryer/moq/blob/github-actions/.github/workflows/test.yml

Failed build: https://github.com/matryer/moq/runs/923049639?check_suite_focus=true

Logs:
```
##[group]Run go get -v -u -t -d ./...
go get -v -u -t -d ./...
shell: /bin/bash -e {0}
env:
  GOPATH: /home/runner/work/moq/moq
  GO111MODULE: off
  GOROOT: /opt/hostedtoolcache/go/1.14.4/x64
##[endgroup]
github.com/matryer/moq (download)
# cd /home/runner/work/moq/moq/src/github.com/matryer/moq; git pull --ff-only
From https://github.com/matryer/moq
 * [new branch]      github-actions  -> origin/github-actions
 * [new branch]      import-aliases  -> origin/import-aliases
 * [new branch]      master          -> origin/master
 * [new branch]      no-global-mutex -> origin/no-global-mutex
 * [new branch]      variadic-args-5 -> origin/variadic-args-5
You are not currently on a branch.
Please specify which branch you want to merge with.
See git-pull(1) for details.

    git pull <remote> <branch>

package github.com/matryer/moq: exit status 1
get "golang.org/x/tools/go/packages": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/go/packages?go-get=1
get "golang.org/x/tools/go/packages": verifying non-authoritative meta tag
golang.org/x/tools (download)
get "golang.org/x/tools/go/gcexportdata": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/go/gcexportdata?go-get=1
get "golang.org/x/tools/go/gcexportdata": verifying non-authoritative meta tag
get "golang.org/x/tools/go/internal/gcimporter": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/go/internal/gcimporter?go-get=1
get "golang.org/x/tools/go/internal/gcimporter": verifying non-authoritative meta tag
get "golang.org/x/tools/go/internal/packagesdriver": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/go/internal/packagesdriver?go-get=1
get "golang.org/x/tools/go/internal/packagesdriver": verifying non-authoritative meta tag
get "golang.org/x/tools/internal/gocommand": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/internal/gocommand?go-get=1
get "golang.org/x/tools/internal/gocommand": verifying non-authoritative meta tag
get "golang.org/x/mod/semver": found meta tag get.metaImport{Prefix:"golang.org/x/mod", VCS:"git", RepoRoot:"https://go.googlesource.com/mod"} at //golang.org/x/mod/semver?go-get=1
get "golang.org/x/mod/semver": verifying non-authoritative meta tag
golang.org/x/mod (download)
get "golang.org/x/tools/internal/event": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/internal/event?go-get=1
get "golang.org/x/tools/internal/event": verifying non-authoritative meta tag
get "golang.org/x/tools/internal/event/core": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/internal/event/core?go-get=1
get "golang.org/x/tools/internal/event/core": verifying non-authoritative meta tag
get "golang.org/x/tools/internal/event/keys": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/internal/event/keys?go-get=1
get "golang.org/x/tools/internal/event/keys": verifying non-authoritative meta tag
get "golang.org/x/tools/internal/event/label": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/internal/event/label?go-get=1
get "golang.org/x/tools/internal/event/label": verifying non-authoritative meta tag
get "golang.org/x/tools/internal/packagesinternal": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/internal/packagesinternal?go-get=1
get "golang.org/x/tools/internal/packagesinternal": verifying non-authoritative meta tag
get "golang.org/x/tools/internal/typesinternal": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/internal/typesinternal?go-get=1
get "golang.org/x/tools/internal/typesinternal": verifying non-authoritative meta tag
get "golang.org/x/xerrors": found meta tag get.metaImport{Prefix:"golang.org/x/xerrors", VCS:"git", RepoRoot:"https://go.googlesource.com/xerrors"} at //golang.org/x/xerrors?go-get=1
golang.org/x/xerrors (download)
get "golang.org/x/xerrors/internal": found meta tag get.metaImport{Prefix:"golang.org/x/xerrors", VCS:"git", RepoRoot:"https://go.googlesource.com/xerrors"} at //golang.org/x/xerrors/internal?go-get=1
get "golang.org/x/xerrors/internal": verifying non-authoritative meta tag
get "golang.org/x/tools/imports": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/imports?go-get=1
get "golang.org/x/tools/imports": verifying non-authoritative meta tag
get "golang.org/x/tools/internal/imports": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/internal/imports?go-get=1
get "golang.org/x/tools/internal/imports": verifying non-authoritative meta tag
get "golang.org/x/mod/module": found meta tag get.metaImport{Prefix:"golang.org/x/mod", VCS:"git", RepoRoot:"https://go.googlesource.com/mod"} at //golang.org/x/mod/module?go-get=1
get "golang.org/x/mod/module": verifying non-authoritative meta tag
get "golang.org/x/tools/go/ast/astutil": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/go/ast/astutil?go-get=1
get "golang.org/x/tools/go/ast/astutil": verifying non-authoritative meta tag
get "golang.org/x/tools/internal/gopathwalk": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/internal/gopathwalk?go-get=1
get "golang.org/x/tools/internal/gopathwalk": verifying non-authoritative meta tag
get "golang.org/x/tools/internal/fastwalk": found meta tag get.metaImport{Prefix:"golang.org/x/tools", VCS:"git", RepoRoot:"https://go.googlesource.com/tools"} at //golang.org/x/tools/internal/fastwalk?go-get=1
get "golang.org/x/tools/internal/fastwalk": verifying non-authoritative meta tag
github.com/pmezard/go-difflib (download)
##[error]Process completed with exit code 1.
```

**Comment by zllovesuki** [comment]
Might have something to do with `go mod download` writes to stderr...

**Comment by sudo-suhas** [comment]
That's not it. Tried with only `go get -u -t ./...` and it still failed - https://github.com/matryer/moq/runs/988481983?check_suite_focus=true

**Comment by sudo-suhas** [comment]
Here you go: https://github.com/matryer/moq/pull/129, https://github.com/matryer/moq/runs/4053234433?check_suite_focus=true

**Comment by Sergey-Murtazin** [comment]
Hi, @sudo-suhas!

I've tried to reproduce the issue on a fork of your repository

But `Go Get` step finished successfully and it failed later.

As you can see here
https://github.com/Sergey-Murtazin/moq/runs/4149826700?check_suite_focus=true

**Comment by sudo-suhas** [comment]
I reported the issue months ago, and after not receiving any response, migrated the project to Go modules. On your request for a public repo where the failure occurred, I did so and the issue still seemed to be present. I do not know why it failed there and why it passed on your fork. At this point, I do not have much interest in getting to the bottom of the issue as this is not an issue I am likely to face again. Feel free to investigate the issue further or close it.
